packages/web/src/components/chat-listener/ChatListener.tsx

```diff
@@ -0,0 +1,25 @@
+import { useEffect } from 'react'
+
+import { Status, accountSelectors, chatActions } from '@audius/common'
+import { useDispatch, useSelector } from 'react-redux'
+
+const { connect, disconnect, fetchMoreChats, fetchUnreadMessagesCount } =
+  chatActions
+const { getAccountStatus } = accountSelectors
+
+export const ChatListener = () => {
+  const dispatch = useDispatch()
+  const accountStatus = useSelector(getAccountStatus)
+  // Connect to chats websockets and prefetch chats
+  useEffect(() => {
+    if (accountStatus === Status.SUCCESS) {
```

Comment:
Any risk on dispatching these actions multiple times in the case of sign out + sign in again in same session?

Reply:
Yes, but it disconnects in the effect cleanup

Reply:
duh